t/more-bundle.t

```diff
@@ -0,0 +1,264 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Deep;
+use JSON::Validator;
+
+# these are triples:
+# - schema name to extract from schema file(s) with bundle()
+# - expected result
+# - test name.
+my @tests = (
```

Comment:
I'm pretty sure none of the files are run through perltidy.

Reply:
correct.

Reply:
Let me rephrase: All the code need to be run through "perltidy" to be accepted. You can use https://metacpan.org/pod/distribution/App-githook-perltidy/bin/githook-perltidy#githook-perltidy-install-[-force,-f] to do this automatically and locally for this repo.